src/datasources/notifications/notifications.datasource.ts

```diff
@@ -0,0 +1,247 @@
+import { NotificationType as DomainNotificationType } from '@/domain/notifications/entities-v2/notification-type.entity';
+import { Uuid } from '@/domain/notifications/entities-v2/uuid.entity';
+import { IAccountsDatasource } from '@/domain/interfaces/accounts.datasource.interface';
+import { INotificationsDatasource } from '@/domain/interfaces/notifications.datasource.interface';
+import { LoggingService, ILoggingService } from '@/logging/logging.interface';
+import { asError } from '@/logging/utils';
+import {
+  Inject,
+  Injectable,
+  NotFoundException,
+  UnprocessableEntityException,
+} from '@nestjs/common';
+import postgres from 'postgres';
+import { UpsertSubscriptionsDto } from '@/datasources/notifications/entities/upsert-subscriptions.dto.entity';
+
+@Injectable()
+export class NotificationsDatasource implements INotificationsDatasource {
+  constructor(
+    @Inject('DB_INSTANCE')
+    private readonly sql: postgres.Sql,
+    @Inject(LoggingService)
+    private readonly loggingService: ILoggingService,
+    @Inject(IAccountsDatasource)
+    private readonly accountsDatasource: IAccountsDatasource,
```

Comment:
Nit: this is not needed now, right?

Reply:
Removed in d49aca2e6a097d8ac438fa5876d39f1b2006bf3f.